test/explorer.spec.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-dom/test-utils';
+import Explorer from '../src/scripts/components/explorer/Explorer';
+
+const moment = require('moment');
+const _ = require('underscore');
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const Promise = require('es6-promise').Promise;
```

Comment:
Take a look at `search.spec.js`: `import sinonStubPromise from 'sinon-stub-promise';` in the latest master and see if that works for you instead of creating the promise manually each time.